src/pkg/path/path.go

```diff
@@ -96,6 +99,25 @@ type Path interface {
 	ToBuilder() *Builder
 }
 
+// MaybeShortRef takes a stirng and returns true if this string could possibly
+// be a ShortRef. Otherwise it returns false. A return value of true does not
+// absolutely mean the string is a ShortRef returned by some item, only that it
+// fits the pattern ShortRefs use.
```

Comment:
```suggestion
// MaybeShortRef takes a string and returns true if this string complies with
// the ShortRef format. Otherwise it returns false. A return value of true does not
// guarantee that the string is a ShortRef, nor that it represents an existing item.
```